src/commonMain/kotlin/org/kosat/Clause.kt

```diff
@@ -0,0 +1,35 @@
+package org.kosat
+
+typealias Lit = Int
+
+class Clause(val lits: MutableList<Lit>): Collection<Lit> {
```

Comment:
If you REALLY WANT to inherit Collection, you could use [`by-delegation`](https://kotlinlang.org/docs/delegation.html#overriding-a-member-of-an-interface-implemented-by-delegation):
```kotlin
class Clause(val lits: MutableList<Lit>): Collection<Lit> by lits {
    ...
}
```